### ISSUE TITLE
JsDbg Fails to launch in VS2019

### ISSUE THREAD

**Issue by SamFort-MSFT** [MicrosoftEdge/JsDbg#136]
When trying to load JsDbg in VS2019, I get the following error:

<record>2457</record>
<time>2019/05/23 00:57:20.350</time>
<type>Error</type>
<source>VisualStudio</source>
<description>LegacySitePackage failed for package [JsDbgPackage]Source: &apos;JsDbg.VisualStudio&apos; Description: Object reference not set to an instance of an object.&#x000D;&#x000A;System.NullReferenceException: Object reference not set to an instance of an object.&#x000D;&#x000A;   at JsDbg.VisualStudio.AutoUpdater.CheckForUpdates(String identifier, String updateUrl)&#x000D;&#x000A;   at JsDbg.VisualStudio.JsDbgPackage.Initialize()&#x000D;&#x000A;   at Microsoft.VisualStudio.Shell.Package.Microsoft.VisualStudio.Shell.Interop.IVsPackage.SetSite(IServiceProvider sp)</description>
<guid>{5B3AF206-B4D4-4D12-9661-5D2D8DD8D194}</guid>
<hr>80004003 - E_POINTER</hr>
<errorinfo></errorinfo>

**Comment by cbiesinger** [comment]
I saw the same thing on vs2017, I mentioned it to Sanket

**Comment by sanketj** [comment]
Yes, I've been working on a fix for the last couple of days but haven't been able to get to the root cause yet.  I'm planning to revert the regressing change today to unblock VS 2017 installs, and then continue investigating the problem.

**Comment by sanketj** [comment]
I have reverted the regressing change and uploaded a fixed vsix. Keeping this issue open and renaming it to track the re-enablement of VS 2019 support.
